frontend/modules/common/Api.js

```diff
@@ -0,0 +1,56 @@
+import request from 'superagent';
```

Comment:
Yep thats what I'm thinking. I know that moving all the apis in there is out of scope right now. But should be move this file to something like `api/BrowseApi.js`